### PR TITLE
Add text to image loading state

### PR DESCRIPTION
**Pull request by dmitryvlass**
### Description

In this PR, I am adding the T2I loading state, keeping it in the localstorage.

https://user-images.githubusercontent.com/121915729/212230533-cf8e82aa-72a5-443d-97cb-4831ee83f6e6.mp4

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -14,6 +14,9 @@ import {
 } from "components";
 import { vanaPost } from "vanaApi";
 
+// Number of "text to images" generated per request.
+const imagesPerRequest = 8;
```
Comment (dmitryvlass) [inline pages/index.js:18]: @tnunamak , could you help to make sure the const is correct?

Comment (callumflack) [reply pages/index.js:18]: @dmitryvlass since Tim's on holidays, we may have to confirm this after merging.

Comment (tnunamak) [reply pages/index.js:18]: I'm on my phone atm but it should match whatever the API request passes as an option. Shouldn't be far...

Comment (dmitryvlass) [reply pages/index.js:18]: Ah, thanks. I haven't noticed that we are passing this number in the variable. I just updated my PR to expose this const.

File: components/prompt/Prompt.module.css [hunk 2]
```diff
@@ -19,6 +19,29 @@
...
+@keyframes placeHolderShimmer {
+  0% {
+    background-position: -300px 0
+  }
+  100% {
+    background-position: 300px 0
+  }
+}
+
+.galleryImageLoading {
```
Comment (callumflack) [inline components/prompt/Prompt.module.css:31]: @dmitryvlass I'd like to do something more to clarify this UI state. But I'd like to do that in [VAN-532](https://linear.app/vana-team/issue/VAN-532/upgrade-the-visual-design).

File: styles/Home.module.css [hunk 3]
```diff
@@ -14,8 +14,8 @@
   justify-content: center;
   /* align-items: center; */
   position: relative;
-  padding-top: 7vh;
```
Comment (callumflack) [inline styles/Home.module.css:17]: @dmitryvlass the 7vh looks better IMHO. I didn't want to bother revising the entire spacing variables to move this value into it… But if you insist, then I will. Can do in [VAN-532](https://linear.app/vana-team/issue/VAN-532/upgrade-the-visual-design)?

File: styles/globals.css [hunk 4]
```diff
@@ -96,6 +96,12 @@ h1,
 h2,
 h3 {
   line-height: 1.1;
+  font-family: var(--font-display);
```
Comment (callumflack) [inline styles/globals.css:99]: @dmitryvlass the Spectral serif has tracking that is too wide IMHo. But again, can fix in [VAN-532](https://linear.app/vana-team/issue/VAN-532/upgrade-the-visual-design).

**Review by callumflack — APPROVED** [review]
@dmitryvlass coupla very minor comments but this is really nice and ready to go. I personally don't intend to spend much more time on this as I've got design work to do for the new chat app.

File: styles/Home.module.css (hunk 3 above)
Comment (dmitryvlass) [reply styles/Home.module.css:17]: Yeah, I don't want to use viewport relative units, as the behaviours on portrait/landscape will be different. 7vh on mobile is way bigger than 7vh on desktop.

File: styles/globals.css (hunk 4 above)
Comment (dmitryvlass) [reply styles/globals.css:99]: Yeah, I just wanted to put a Serif font for headings. Feel free to make it prettier

**Pull request merged**